### ISSUE TITLE
py3: review v4l bindings

### ISSUE THREAD

**Issue by JohnDMcMaster** [JohnDMcMaster/pyuscope#12]
The v4l library has been abandoned and is bit-rotting under python3. Review the various forks and try to coordinate one to be the official fork and/or at least advocate one

**Comment by JohnDMcMaster** [comment]
TLDR: upgrade to https://github.com/antmicro/python3-v4l2

Details:
https://launchpad.net/python-v4l2
  * Python3: no
  * Last update: 2018-03-03
  * Original abandoned project I've been using
  * https://pypi.org/project/v4l2/

https://github.com/antmicro/python3-v4l2
  * Python3: yes
  * Last update: Jul 29, 2019
  * Stars: 3
  * Derived from launchpad repo
  * Quick test looks like it works

https://github.com/AlexJinlei/python-v4l2
  * Python3: no
  * Last update: Jun 12, 2017
  * Stars: 8
  * Derived from launchpad repo
  * Claims it has some v4l2 updates

https://github.com/duanhongyi/pyv4l2
  * Python3: yes
  * Last update: May 15, 2019
  * Looks more comprehensive (ex: maybe easier to get frames), but its different API than existing code and I don't need to get frames (only set controls)
  * Got C code build failure from source, but pip install worked
